services/QuillLMS/config/storage.yml

```diff
@@ -0,0 +1,14 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
+amazon:
+  service: S3
+  access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  region: <%= ENV['AWS_REGION'] || 'us-east-1' %>
+  bucket: <%= ENV['AWS_S3_BUCKET'] || 'quill-lms-uploads' %>
```

Comment:
Since we use other S3 buckets in LMS, let's make this name more specific. `AWS_S3_UPLOADS_BUCKET`, for example

Reply:
That's a good point.  Updated